packages/store-cache/src/utils.test.ts

```diff
@@ -0,0 +1,130 @@
+import { beforeEach, describe, expect, it, vi } from "vitest";
+import { createDatabase, createDatabaseClient } from ".";
+import { mudConfig } from "@latticexyz/store/register";
```

Comment:
rule of thumb is to use it only if you need `mudConfig` (which is the case here). It doesn't even export much else so you can't use it interchangeably with `"@latticexyz/store"` anyway

Reply:
but why wouldn't I use `mudConfig` from `@latticexyz/config`?

Reply:
after https://github.com/latticexyz/mud/pull/752 is merged, `@latticexyz/config` won't export `mudConfig`, so you can't. Because `mudConfig` depends on global types which kinda break if you don't register them properly

Reply:
and `@latticexyz/config` exports only `mudCoreConfig`, which is different and shouldn't really be used outside of mud internals

Reply:
I have lost the plot